Example/iOS/ViewControllers/SampleListViewController.swift

```diff
@@ -83,13 +83,28 @@ final class SampleListViewController: CollectionViewController {
       ]
     }
 
+    if isTopLevel {
+      items += [
+        LinkView.itemModel(
+          dataID: "Controls Demo",
+          content: .init(animationName: nil, title: "Controls Demo"))
+          .didSelect { [weak self] context in
+            self?.show(ControlsDemoViewController(), sender: context.view)
+          },
+      ]
+    }
```

Comment:
nit: Suggest breaking this out into multiple methods that return arrays of items models for readability

Reply:
Good suggestion, much better now https://github.com/airbnb/lottie-ios/pull/1462/commits/48f98df7802f3b9a5693275bb06088815cc00968